### DISCUSSION TITLE
streaming server says "self signed certificate in certificate chain"

### DISCUSSION THREAD

**Discussion by thefinn93** [mastodon/mastodon#19917] [General]
I recently updated my Mastodon server from 2.8.4 to 3.5.5. I did the migrations and it generally seems to be working, except the streaming server which appears to be 401ing all requests, printing this error message:

```
ERR! Error: self signed certificate in certificate chain
```

What certificate chain is it talking about? Is it trying to connect to redis with TLS? Or something else? I'm not really sure how to even begin to debug this.

**Comment by thefinn93** [comment]
maybe it's the database? does postgres with sslmode=require validate certs now?

**Comment by thefinn93** [comment]
After searching around, I ended up added environment variable `NODE_TLS_REJECT_UNAUTHORIZED=0` which seems scary and sub-optimal, but makes it work

**Comment by zeeZ** [comment]
Related issue: #11445
